components/proxy/src/main/java/com/hotels/styx/infrastructure/FileBackedRegistry.java

```diff
@@ -118,6 +136,10 @@ private boolean hasFileContentChanged() {
         }
     }
 
+    private String readFileAsString() {
+        return new String(readFile());
+    }
+
     /**
      * Reader.
      *
```

Comment:
The *Reader* interface should really be called *Parser*. It doesn't really read anything, just parses (or deserialises) an array of bytes into objects of *T*.